client/src/components/Navbar/Menus/Authenticated/List.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import {
+  makeStyles,
+  Theme,
+  createStyles,
+  List,
+  ListItem,
+  ListItemIcon,
+  ListItemText,
+  Collapse,
+  Avatar,
+} from '@material-ui/core';
+import ExpandLess from '@material-ui/icons/ExpandLess';
+import ExpandMore from '@material-ui/icons/ExpandMore';
+import StarBorder from '@material-ui/icons/StarBorder';
+import { IoNotificationsSharp } from 'react-icons/io5';
+import { useHistory } from 'react-router-dom';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      minWidth: 150,
+      maxWidth: 360,
+      backgroundColor: theme.palette.background.paper,
+      paddingLeft: theme.spacing(0),
+    },
+    nested: {
+      paddingLeft: theme.spacing(4),
+    },
+    icons: {
+      width: 10,
+      justifyContent: 'flex-end',
+    },
+    none: {
+      color: theme.palette.background.paper,
+    },
+    text: {
+      minWidth: 90,
+      textAlign: 'right',
+      justifySelf: 'center',
+    },
+    button: {
+      // justifyContent: 'space-between',
+    },
+  }),
+);
+
+const NestedList = (): JSX.Element => {
+  const classes = useStyles();
+  const [open, setOpen] = React.useState<boolean>(false);
+  const history = useHistory();
```

Comment:
i suggest moving this line above the states, just to stay clean and maintainable :)